docs/docs/reference/features-classification.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: doc-page
+title: A Classification of Proposed Language Features
+date: February 28, 2019
+author: Martin Odersky
+---
+
+This document provides an overview of the features proposed for Scala 3 with the aim to facilitate the discussion what to include and when to include it. It classifies features into seven groups: Essential foundations, simplifications, restrictions, dropped features, changed features, new features, and features oriented towards meta-programming with the aim to replace existing macros.
```

Comment:
It looks like it's now eight feature groups, not seven since "Changes to Type Checking and Inference" is included.